.travis.yml

```diff
@@ -46,9 +48,13 @@ env:
         # E902 - IOError
         - FLAKE8_OPT="--select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902"
 
-    matrix:
-        - PYTHON_VERSION=3.5 SETUP_CMD='egg_info'
-        - PYTHON_VERSION=3.6 SETUP_CMD='egg_info'
+stages:
+   # Do the style check and a single test job, don't proceed if it fails
+   - name: single_test
+   # Do the rest of the tests
+   - name: tests
+   - name: cron_only
```

Comment:
Can stage names include spaces? If so, I would suggest: "Initial tests", "Comprehensive tests", "Cron tests". If spaces are not allowed, I would recommend the same but with underscores.

Reply:
That's a good question. I'll try out, your naming scheme is way much better!